test/e2e/specs/block-deletion.test.js

```diff
@@ -42,7 +42,7 @@ describe( 'block deletion -', () => {
 
 	describe( 'deleting the third block using the Remove Block shortcut', () => {
 		it( 'results in two remaining blocks and positions the caret at the end of the second block', async () => {
-			await pressWithModifier( [ 'Shift', META_KEY ], 'x' );
+			await pressWithModifier( [ 'Shift', META_KEY ], 'Backspace' );
```

Comment:
Looks like the e2e tests because this was not updated. Not sure when this shortcut changed but was left as is in the e2e tests 

cc @talldan @jasmussen 

This is the right shortcut to delete the block, right?

Reply:
Interesting. Yeah would be nice to have @talldan answer here. I know that the backspace keyboard combo needs to be changed, but a different change was also merged in and later reverted. Perhaps this PR was created before the latter change was reverted? I'm pretty sure it's Backspace in master.

Reply:
yes, I checked the code and it's backspace, and I saw the revert commit but for some reason the revert didn't revert the e2e test and it wasn't failing in e2e tests. Maybe the e2e test was added in parallel.

Reply:
@youknowriad Sorry about that, the test was added in parallel, but I missed reverting it back before flying out to our meetup. Here's a PR that should fix things:
https://github.com/WordPress/gutenberg/pull/9477